ush/forecast_postdet.sh

```diff
@@ -162,19 +163,40 @@ EOF
         delimiter=""
         IAU_INC_FILES=""
         for iaufhr in "${iaufhrs[@]}"; do
-          if (( iaufhr == 6 )); then
-            inc_file="atminc.nc"
+          if [[ "${DO_JEDIATMVAR:-NO}" == "YES" ]]; then
+            for tile in {1..6}; do
+              if (( iaufhr == 6 )); then
+                inc_file="atminc.tile${tile}.nc"
+              else
+                inc_file="atmi$(printf %03i "${iaufhr}").tile${tile}.nc"
+              fi
+              inc_files+=("${inc_file}")
+              IAU_INC_FILES="${IAU_INC_FILES}${delimiter}'${inc_file}'"
+            done
           else
-            inc_file="atmi$(printf %03i "${iaufhr}").nc"
+            if (( iaufhr == 6 )); then
```

Comment:
Not a request to change since this was already in place, but just a note.  I generally prefer `[[ ]]` logic syntax over `(( ))` as the latter, when logged, will appear as
`(( iaufhr == 6 ))`
whereas the former will appear as
`[[ 6 -eq 6 ]]`
which is more informative when determining which branch of an if block, etc, should go down.  I think NCO would prefer it this way as well.